src/prompts/component-confirm.js

```diff
@@ -0,0 +1,30 @@
+const getName = require("../prompts/component-name");
+const questions = require('./questions');
+
+const ask = async (answers, inquirer) => {
+	// TODO: use a different logger
```

Comment:
Hmm, are you wanting something like the [debug](https://www.npmjs.com/package/debug) module?

Reply:
🤔 maybe! But this is more of a detail that we can revisit when the core of the command is set up 👍